lib/types/monday-api-client.ts

```diff
@@ -0,0 +1,60 @@
+
+export type APIOptions = {
+    /**
+     * Access token for the API
+     * If not set, will use the credentials of the current user (client only)
+     */
+    token?: string;
+
+    /**
+     * An object containing GraphQL query variables
+     */
+    variables?: object;
+
+    /**
+     * A string specifying which version of the API should be used
+     * If not set, will use the current API version
+     */
+    apiVersion?: string;
+}
+
+export type APIData = {
+    query: string;
+    variables?: object;
+}
+
+export type APIVersion = string;
+
+export type APIResponse = {
```

Comment:
that's not always the case, what about errors?

Reply:
Look at `lines 35 ~ 37`.
@ShayElkana Why are these types separated in this manner? Developers can get errors, no?